cdc/capture/capture.go

```diff
@@ -732,3 +754,8 @@ func (c *captureImpl) StatusProvider() owner.StatusProvider {
 	}
 	return owner.NewStatusProvider(c.owner)
 }
+
+// IsReady returns if the cdc server is ready
```

Comment:
Comments should also be added to the interface.

Reply:
done